libs/payments/ui/src/lib/client/components/CouponForm/index.tsx

```diff
@@ -67,23 +68,23 @@ const WithoutCoupon = ({
   cartVersion,
   readOnly,
 }: WithoutCouponProps) => {
-  async function applyCoupon(formData: FormData) {
+  async function applyCoupon(prevState: any, formData: FormData) {
```

Comment:
nit. Since it's not being used by the function, could `prevState` be replaced with`_` instead so that it gets rid of the eslint warning?

Reply:
TIL